starters/fastapi-safir-uws/values.yaml

```diff
@@ -0,0 +1,232 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+config:
+  # -- Choose from the text form of Python logging levels
+  loglevel: "INFO"
+
+  # -- Logging profile (`production` for JSON, `development` for
+  # human-friendly)
+  logProfile: "production"
+
+  # -- URL path prefix for the API
+  pathPrefix: "/api/<CHARTNAME>"
+
+  # -- URL for the PostgreSQL database if Cloud SQL is not in use
+  # @default -- None, must be set if `cloudsql.enabled` is false
+  databaseUrl: null
+
+  # -- Grace period in seconds to wait for worker jobs to finish
+  gracePeriod: 60
+
+  # -- Lifetime of job results in Safir `parse_timedelta` format
+  lifetime: "30d"
+
+  # -- Google service account with an IAM binding to the Kubernetes service
+  # account for the application. Must have the `cloudsql.client` role, access
+  # to write to the GCS bucket, and ability to sign URLs as itself
+  # @default -- None, must be set
+  serviceAccount: null
+
+  # -- URL for the GCS bucket for results (must start with `s3` or `gs`)
+  # @default -- None, must be set
+  storageBucketUrl: null
+
+  # -- Timeout for results from a sync request in Safir `parse_timedelta`
+  # format
+  syncTimeout: "1m"
+
+  # -- Timeout for a single job in seconds
+  # @default -- 600 (10 minutes)
+  timeout: 600
+
+image:
+  # -- Image to use for the frontend and database workers
+  repository: "ghcr.io/lsst-sqre/<CHARTNAME>"
+
+  # -- Pull policy for the image
+  pullPolicy: "IfNotPresent"
+
+  # -- Tag of image to use
+  # @default -- The appVersion of the chart
+  tag: null
+
+ingress:
+  # -- Additional annotations to add to the ingress
+  annotations: {}
+
+frontend:
+  # -- Number of web frontend pods to start
+  replicaCount: 1
+
+  # -- Resource limits and requests for the <CHARTNAME> frontend pod
+  # @default -- See `values.yaml`
+  resources: {}
+
+  # -- Affinity rules for the <CHARTNAME> frontend pod
+  affinity: {}
+
+  # -- Node selector rules for the <CHARTNAME> frontend pod
+  nodeSelector: {}
+
+  # -- Annotations for the <CHARTNAME> frontend pod
+  podAnnotations: {}
+
+  # -- Tolerations for the <CHARTNAME> frontend pod
+  tolerations: []
+
+cloudsql:
+  # -- Enable the Cloud SQL Auth Proxy sidecar, used with Cloud SQL databases
+  # on Google Cloud
+  enabled: false
+
+  image:
+    # -- Cloud SQL Auth Proxy image to use
+    repository: "gcr.io/cloudsql-docker/gce-proxy"
+
+    # -- Cloud SQL Auth Proxy tag to use
+    tag: "1.35.4"
+
+    # -- Pull policy for Cloud SQL Auth Proxy images
+    pullPolicy: "IfNotPresent"
+
+  # -- Instance connection name for a Cloud SQL PostgreSQL instance
+  # @default -- None, must be set if Cloud SQL is used
+  instanceConnectionName: null
+
+  # -- Resource limits and requests for the Cloud SQL Proxy container
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "100m"
+      memory: "20Mi"
+    requests:
+      cpu: "5m"
+      memory: "7Mi"
+
+cutoutWorker:
```

Comment:
Seems like this should be generic, like "worker" or "pipelinesWorker" ?

Reply:
Thank you, good catch. I grepped for `cutouts` and not `cutout`. Found and fixed all of these, I think.